public/app/plugins/panel/table/module.tsx

```diff
@@ -113,6 +113,16 @@ export const plugin = new PanelPlugin<Options, FieldConfig>(TablePanel)
           name: t('table.name-wrap-text', 'Wrap text'),
           category,
         })
+        .addNumberInput({
+          path: 'maxHeight',
+          name: t('table.text-wrap-options.label-max-height', 'Max cell height'),
+          category,
+          settings: {
+            placeholder: t('table.text-wrap-options.placeholder-max-height', 'none'),
+            min: 0,
```

Comment:
wonder if we should add an internal minimum so you can't set it to like 1px. realistically this shouldnt be less than the height of a single line?